iron-doc-behavior.html

```diff
@@ -150,11 +171,16 @@
         return element.classname || element.tagname;
       },
 
-      _compareElements: function(a, b) {
-        if (a.tagname === b.tagname) {
-          return a.classname == null ? -1 : a.classname.localeCompare(b.classname);
-        }
-        return a.tagname == null ? -1 : b.tagname == null ? 1 : a.tagname.localeCompare(b.tagname);
+      /**
+       * Compare two analysis descriptors (elements, functions, etc.) by
+       * display name.
+       */
+      _compareDescriptors: function(a, b) {
+        // Elements display as "<tagname> (classname)" or "classname", while
+        // everything else displays as "name". Sort according to that format.
+        var aName = (a.name || '') + (a.tagname || '') + (a.classname || '');
+        var bName = (b.name || '') + (b.tagname || '') + (b.classname || '');
```

Comment:
I don't want to block on this, but this will still cause comparing classname against tagname in some cases, where I was hoping to put all elements with a tagname together, all without together, etc. What I really want is an `orderBy('name', 'tagname', 'class')` semantics.